File: EmployeeMS/EmployeeMS.Domain/DTOs/Employee/GetEmployeeDTO.cs
using EmployeeMS.Domain.DTOs.EmployeeContract;
using EmployeeMS.Domain.DTOs.EmployeeFile;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EmployeeMS.Domain.DTOs.Employee
{
    public class GetEmployeeDTO
    {
        public int? Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public DateOnly DateOfBirth { get; set; }
        public string JobTitle { get; set; }
        public int DepartmentId { get; set; }
        public DateOnly HireDate { get; set; }
        public ICollection<GetEmployeeFileDTO> EmployeeFiles { get; set; }
        public ICollection<GetEmployeeContractDTO> EmployeeContracts{ get; set; }
    }
}

File: EmployeeMSClient/src/app/core/DTOs/employee/GetEmployeeDTO.ts
export interface GetEmployeeDTO {
  id: number;
  name: string;
  email: string;
  phoneNumber: string;
  dateOfBirth: string;
  hireDate: string;
  jobTitle: string;
  departmentId: number;
  employeeFiles?: GetEmployeeFileDTO[];
  profilePicture?: File | null;
}

export interface GetEmployeeFileDTO {
  id: number;
  filePath: any;
  employeeId: number;
  employeeFileTypeId: number;
}
